network-api/networkapi/wagtailpages/pagemodels/blog/blog.py

```diff
@@ -46,14 +49,45 @@ class BlogPageTag(TaggedItemBase):
     content_object = ParentalKey('wagtailpages.BlogPage', on_delete=models.CASCADE, related_name='tagged_items')
 
 
-class BlogPage(FoundationMetadataPageMixin, Page):
+@register_snippet
+class BlogAuthor(models.Model):
 
-    author = models.CharField(
-        verbose_name='Author',
-        max_length=70,
-        blank=False,
+    name = models.CharField(max_length=70, blank=False)
+    image = models.ForeignKey(
+        'wagtailimages.Image',
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
     )
 
+    panels = [
+        FieldPanel("name"),
+        ImageChooserPanel("image"),
+    ]
+
+    def __str__(self):
+        return self.name
+
+
+class BlogAuthorsOrderable(Orderable):
```

Comment:
let's not put `orderable` in the class name: the class definition already makes this clear.